stock_scheduler_assignation_horizon/models/procurement_group.py

```diff
@@ -38,3 +40,24 @@ def _get_moves_to_assign_domain_assignation_limited(self, domain, company):
                 [("date", "<=", max_date)],
             ]
         )
+
+    def get_global_is_moves_assignation_limited(self):
```

Comment:
```suggestion
    def _get_global_is_moves_assignation_limited(self):
```
Should be private

Reply:
Since we don't have any company settings anymore, a shorter version as `_is_assignation_horizon_limited` would also be nice :innocent: